src/bp/core/services/stats/index.ts

```diff
@@ -0,0 +1,125 @@
+import axios from 'axios'
+import LicensingService from 'common/licensing-service'
+import { machineUUID } from 'common/stats'
+import { TYPES } from 'core/types'
+import { inject, injectable } from 'inversify'
+import ms from 'ms'
+import path from 'path'
+
+import { GhostService } from '..'
+import { BotService } from '../bot-service'
+import { JobService } from '../job-service'
+
+const LOCK_RESOURCE = 'botpress:statsService'
+@injectable()
+export class StatsService {
+  constructor(
+    @inject(TYPES.JobService) private jobService: JobService,
+    @inject(TYPES.BotService) private botService: BotService,
+    @inject(TYPES.GhostService) private ghostService: GhostService,
+    @inject(TYPES.LicensingService) private licenseService: LicensingService
+  ) {}
+
+  public start() {
+    // tslint:disable-next-line: no-floating-promises
+    this.run()
+    setInterval(this.run.bind(this), ms('6 hours'))
+  }
+
+  private async run() {
+    const lock = await this.jobService.acquireLock(LOCK_RESOURCE, ms('1 minute'))
+    if (lock) {
+      await this.sendStats()
+    }
+  }
```

Comment:
Your interval runs for 6 hours but you keep the lock for only 1 minute. Usually your lock needs to be longer than your interval, otherwise it has no use. If you start 3 nodes, 1 hour apart, they will all send their report